canonical/canonical.com
# WD-8178 - get in touch forms updated

## Done

Update get in touch forms to open when a link too the page has a hash

## QA

- [demo link](https://canonical-com-1158.demos.haus/)
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that forms are working correctly on all pages

## Issue / Card
[WD-8178](https://warthogs.atlassian.net/browse/WD-8178)

Fixes #

## Screenshots



[WD-8178]: https://warthogs.atlassian.net/browse/WD-8178?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
